From 7e5174e3123c3e021f2569e115400f1d68972736 Mon Sep 17 00:00:00 2001
From: Dominggoes Isakh <drjisakh@gmail.com>
Date: Sat, 1 Feb 2020 08:34:04 +0100
Subject: [PATCH] Parcel: HACK: Don't crash when Remove() is called

This fixes RIL for i9100. I need to figure out where this is called.

Change-Id: I9d04a1d40b63b71d93dcf22861a3a9f81d747d24
---
 Parcel.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/system/libhwbinder/Parcel.cpp b/system/libhwbinder/Parcel.cpp
index d6ea4813..1d2b1de6 100644
--- a/system/libhwbinder/Parcel.cpp
+++ b/system/libhwbinder/Parcel.cpp
@@ -1098,7 +1098,7 @@ status_t Parcel::writeEmbeddedNativeHandle(const native_handle_t *handle,
 
 void Parcel::remove(size_t /*start*/, size_t /*amt*/)
 {
-    LOG_ALWAYS_FATAL("Parcel::remove() not yet implemented!");
+    ALOGE("Parcel::remove() not yet implemented!");
 }
 
 status_t Parcel::read(void* outData, size_t len) const
